index.html

```diff
@@ -1,82 +1,69 @@
-    <link rel="stylesheet" href="test.css" />
-    <title>CAkE Portfolio</title>
-  </head>
 
-  <body>
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <script src="https://kit.fontawesome.com/0728e06898.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="style.css" />
+  <link rel="stylesheet" href="test.css" />
+  <title>CAkE Portfolio</title>
+</head>
 
-    <main class="main">
-        <header class="header">
-            <a href="#" class="my-logo">CAkE</a>
-            <nav class="nav-bar">
-              <span class="mob-menu"
-                ><img src="Images&Icons/Enabled.png" alt=""
-              /></span>
-            </nav>
-          </header>
-      <section class="banner">
-        <!-- <img class="banner-img" src="Images&Icons/mobile shapes.png" alt=""> -->
-        <div class="intro-text">
-          <h1>
-            I'm CAkE <br />
-            Glad to see <br />
-            you!
-          </h1>
-          <p>
-            i'm software developer! i can help you build a product, feature or
-            website look through some of my work and experience! if you like
-            what you see and have a project you need coded, don't hesitate to
-            contact me.
-          </p>
-        </div>
-        <div class="contact">
-          <a class="connect" href="#">Let's Connect</a>
-          <ul>
-            <li>
-              <a href="#"
-                ><span class="icons"
-                  ><img src="Images&Icons/twitter.png" alt="" /></span
-              ></a>
-            </li>
-            <li>
-              <a href="#"
-                ><span class="icons"
-                  ><img src="Images&Icons/Vector.png" alt="" /></span
-              ></a>
-            </li>
-            <li>
-              <a href="#"
-                ><span class="icons"
-                  ><img src="Images&Icons/github).png" alt="" /></span
-              ></a>
-            </li>
-            <li>
-              <a href="#"
-                ><span class="icons"
-                  ><img src="Images&Icons/yeah.png" alt="" /></span
-              ></a>
-            </li>
-          </ul>
-        </div>
-      </section>
+<body>
 
-      <section class="works">
-        <div class="Projects">
-          <img
-            class="work-banners"
-            src="Images&Icons/SnapshootPortfolio.png"
-            alt="Portfolio image"
-          />
+  <main class="main">
+    <header class="header">
+      <a href="#" class="my-logo">CAkE</a>
+      <nav class="nav-bar">
+        <span class="mob-menu"><img src="Images&Icons/Enabled.png" alt="" /></span>
+        <ul class="desktop-nav">
+          <li><a href="">Portfolio</a></li>
+          <li><a href="">About</a></li>
+          <li><a href="">Contact</a></li>
+        </ul>
+      </nav>
+    </header>
+    <section class="banner">
+

```

Comment:
Kindly fix this problem as directed by previous reviewer, since it is still existing and hasn't been fixed yet.
Your design ⬇️
![image](https://user-images.githubusercontent.com/33268927/177726996-4bb61b9b-92f2-487f-92df-0f3e1df050e8.png)